src/test/java/uk/gov/hmcts/sptribs/common/event/CreateTestCaseTest.java

```diff
@@ -30,6 +32,8 @@ class CreateTestCaseTest {
     @InjectMocks
     private CreateTestCase createTestCase;
 
+    @Mock
+    private CcdSupplementaryDataService coreCaseApiService;
```

Comment:
rename the variable to include SupplementaryData.

Reply:
Done.